### ISSUE TITLE
Update for latest Patch of Stalker 2

### ISSUE THREAD

**Issue by unk0wn-u53r** [alxn1/d3d12-proxy#1]
The Fix doesn't work anymore, I wonder that no one here report this issue :D

**Comment by alxn1** [comment]
Strange :) Maybe it's due to long shaders compilation on start (but i'm not sure). I can suggest to remove this option in ini file: https://github.com/alxn1/d3d12-proxy/blob/cb19325766c0fecd79bcad8aadef0cc86fa4e5c6/dxgi-proxy/dxgi_proxy.ini#L23
Maybe it help.

**Comment by alxn1** [comment]
Or (it will be better) try to change the option value to 120/180/240/1000 (the option is to disable all overrides after N seconds, and if shaders precompilation takes more time than the default 60 seconds on game start, UE5 was perform a GPU check after, and the fix doesn't work).

**Comment by NiutoN83** [comment]
Sorry bro, but fix not work in Stalker2 and Bodycam.
I try redact Line 23 in [cb19325] - no result. Still cant see many shadow...

**Comment by alxn1** [comment]
I can't confirm this :) In S.T.A.L.K.E.R. 2 fix works fine (i did test it itself with all latest patches), about Bodycam i can't say anything. Anyway, maybe i will spend some time later and retest fix, but currently it issue looks like something wrong not from fix side.
I can suggest remove line 23 from .ini file and check that path where you place .dll and .ini file is correct.

**Comment by NiutoN83** [comment]
I moved all the files from the d3d12-proxy.zip archive to the folder where the Stalker2.exe file is located. I have the steam version. updated  was launched not through Steam, but from the folder with the game through the Stalker2.exe file
and I tried an earlier version of your patch. I have a 2023 Rog Ally (white)

**Comment by alxn1** [comment]
No, i think this is not right directory. You should place files to "Stalker2/Binaries/Win64" folder (where Stalker2-Win64-Shipping.exe is placed)
As i understand, Stalker2.exe is something like are launcher, not are game really.

**Comment by NiutoN83** [comment]
YES, IT WORK SHANK YOU
I use version vhen i delite 23 line

**Comment by alxn1** [comment]
Good news! Thank you!

**Comment by NiutoN83** [comment]
I have another question. I solved my problem, but it was very difficult to find you. Can I post your files on Nexus, which is a more accessible resource for many?. Of course, I will link to the source and the original file in the description, but I need your consent for this

**Comment by alxn1** [comment]
Without any problem, you can do it :)
